automata/symbol/graph/symbol_graph.py

```diff
@@ -14,6 +16,9 @@
 
 logger = logging.getLogger(__name__)
 
+default_data_root_path = "automata-embedding-data"
```

Comment:
Let's not set the default here, let's rather get the `DATA_ROOT_PATH` from the automata.config and set the default there, as per previous comments.

Reply:
This is the config/__init__ snippet, for reference 

```python
load_dotenv()

# Define environment variables
OPENAI_API_KEY = os.getenv("OPENAI_API_KEY", "")
GITHUB_API_KEY = os.getenv("GITHUB_API_KEY", "")
CONVERSATION_DB_PATH = os.getenv(
    "CONVERSATION_DB_PATH", os.path.join("..", "conversation_db.sqlite3")
)
EVAL_DB_PATH = os.getenv("EVAL_DB_PATH", os.path.join("..", "eval_db.sqlite3"))
TASK_DB_PATH = os.getenv("TASK_DB_PATH", os.path.join("..", "task_db.sqlite3"))
TASK_OUTPUT_PATH = os.getenv(
    "TASKS_OUTPUT_PATH", os.path.join("..", "local_tasks")
)
REPOSITORY_NAME = os.getenv("REPOSITORY_NAME", "emrgnt-cmplxty/Automata")
MAX_WORKERS = int(os.getenv("MAX_WORKERS", 8))
```